app/utils/Strings.scala

```diff
@@ -38,6 +38,37 @@ object Strings {
     Random.alphanumeric.filter(_.isDigit).take(length).mkString
   }
 
+  def decimalMax12(): String =
+    Seq(
+      Gen.choose(1, 9).sample.getOrElse(1).toString,
+      Gen.choose(1, 9).sample.getOrElse(2).toString,
+      Gen.choose(1, 9).sample.getOrElse(3).toString,
+      Gen.choose(1, 9).sample.getOrElse(4).toString,
+      Gen.choose(1, 9).sample.getOrElse(5).toString,
+      Gen.choose(1, 9).sample.getOrElse(6).toString,
+      Gen.choose(1, 9).sample.getOrElse(7).toString,
+      Gen.choose(1, 9).sample.getOrElse(8).toString,
+      Gen.choose(1, 9).sample.getOrElse(9).toString,
+      Gen.choose(1, 9).sample.getOrElse(3).toString,
+      Gen.choose(1, 9).sample.getOrElse(5).toString,
+    ).mkString
```

Comment:
We've got code that eliminates the need for the getOrElse in the departures backend (see the generators.value defs). Also, why not just do something like the methods above where the length can be provided, and simply use a recursive method to generate the string. (Look up tail recursion helper methods if you need an example)

Reply:
You need the getOrElse as the generator can still fail. The value property on Gen returns an Option type

Reply:
There is no reason to force the use of individual digits, arbitrarily chosen in the getOrElse. I pointed you towards the OptionValues stuff because I'd rather throw an exception if the generator was unable to produce a number between 1 and 9 because whilst the generator can technically generate a None, that shouldn't be possible with this specific generator, and I'd rather know if our generator code was failing as a log of some kind. That way we know something went wrong.